### DISCUSSION TITLE
NATO Nemmera (Can i somehow change the vehicle for cleaning trash)

### DISCUSSION THREAD

**Discussion by SniexDejjtik** [Vdauphin/HeartsAndMinds#1482] [Question and answer] [answered]
**hello i need help** i want to change nemmera to diferrent vehicle that can delete [Destroy] trash?
somebody now how to do it or where i can find the file with this.

**Thx for anyhelp**

**Comment by Vdauphin** [comment, marked as answer]
Hello,

You can add vehicle here https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/ied/deleteLoop.sqf#L31

Cheers

**Reply by SniexDejjtik** [reply]
thx verry much
